lib/CMakeLists.txt

```diff
@@ -636,21 +640,25 @@ if(UFE_FOUND)
 
     if(IS_MACOSX OR IS_LINUX)
         mayaUsd_init_rpath(rpath "${PROJECT_NAME}")
-        mayaUsd_add_rpath(rpath "../..")
+        mayaUsd_add_rpath(rpath "../../..")
```

Comment:
Is this a bug you fixed, i.e. it was simply incorrect previously?

Reply:
Sean made me notice the change in installation directory, which now has a ufe sub-folder, thus the need to adjust the rpath.